tensorflow_datasets/core/features/video_feature.py

```diff
@@ -177,3 +181,42 @@ def encode_example(self, video_or_path_or_fobj):
     else:
       encoded_video = video_or_path_or_fobj
     return super(Video, self).encode_example(encoded_video)
+
+  def repr_html(self, ex: np.ndarray) -> str:
+    """Video are displayed as GIFs."""
+    gif = [_create_thumbnail(frame) for frame in ex]
+
+    # Convert to base64
+    gif_str = utils.get_base64(
+        lambda buff: gif[0].save(buff, format='GIF', save_all=True,
+                                 append_images=gif[1:], duration=500, loop=0))
+
+    # Display HTML
+    return f'<img src="data:image/gif;base64,{gif_str}"  alt="Gif" />'
+
+
+def _create_thumbnail(ex):
```

Comment:
Duplicating code is bad software engineering practice. When you re-use the code, you should make a util function and reuse it across files.